### PR TITLE
Channels presentation: Add overrides for HarperCollins

### PR DESCRIPTION
**Pull request by jprvita**
Fixes: https://github.com/endlessm/kolibri-explore-plugin/issues/555

### CODE REVIEW

**Comment by jprvita** [comment]
This is missing the header background asset, as the one provided was too large (> 2000KB).

**Comment by jprvita** [comment]
I wasn't able to test this yet, but I think the changes are correct and ready.

File: kolibri_explore_plugin/assets/src/customApps.js [hunk 1]
```diff
@@ -276,6 +278,7 @@ export const ChannelCardIcons = {
   '359e048230974c8f80db1a95dc80d544': 'eie-families.png', // eie spanish
   '217c7164c2974a4da9877a20ae38b98e': 'cas.png', // California Academy of Sciences
   '08897e003ea9489eb3d86fc94ba08c21': 'support-ukraine.png',
+  '608fd926be60462aba2feccba6c050c9': 'harper-collins',
```
Comment (manuq) [inline kolibri_explore_plugin/assets/src/customApps.js:281]:
```suggestion
  '608fd926be60462aba2feccba6c050c9': 'harper-collins.png',
```

File: kolibri_explore_plugin/assets/src/customApps.js [hunk 2]
```diff
@@ -219,6 +220,7 @@ export const ChannelCardThumbnails = {
   '359e048230974c8f80db1a95dc80d544': 'eie-families.jpg', // eie spanish
   '217c7164c2974a4da9877a20ae38b98e': 'cas.jpg', // California Academy of Sciences
   '08897e003ea9489eb3d86fc94ba08c21': 'support-ukraine.png',
+  '608fd926be60462aba2feccba6c050c9': 'harper-collins',
```
Comment (manuq) [inline kolibri_explore_plugin/assets/src/customApps.js:223]:
```suggestion
  '608fd926be60462aba2feccba6c050c9': 'harper-collins.png',
```

**Review by manuq — CHANGES_REQUESTED** [review]

**Comment by manuq** [comment]
@jprvita 2 things came up here while testing. I hope you don't mind if I finish this in order to merge and release!
- The references to icon and thumbnail were missing the file extension.
- For a weird reason we need `channel-overrides/harper-collins/styles.scss` to exist even if empty. So if no color overrides were requested we need to add an empty file to revision, otherwise the build fails.

**Pull request merged**